src/Question.scss

```diff
@@ -16,18 +16,18 @@
     margin: .2em;
   }
 
-  .questionIcon {
+  .question-icon, %question-icon {
```

Comment:
This is awesome! Just a stylistic comment - I like keeping placeholder styles in their own block and extending that from the class's block so if the placeholder needs to be shared across more than one class later, it's clearer what belongs to the placeholder, and what belongs to the class.

Reply:
Thanks, I'll try to add this in a future update!